test/integration/migrations_test.rb

```diff
@@ -676,9 +686,11 @@
   end
 
   describe "MySQL" do
+    let(:adapter) { Lotus::Utils.jruby? ? 'mysql' : 'mysql2' }
```

Comment:
Unfortunately not :/

I hate this, but I don't if it is a good ideia to normalize this in the future, every ORM that I had opportunity to work with had this _gap_ between platforms